lib/salus/package_utils/gradle_dependency_parser.rb

```diff
@@ -1,16 +1,11 @@
 module Gradle
-  GRADLE7 = "/opt/gradle/gradle-7.3.3/bin/gradle".freeze
-  GRADLE6 = "/opt/gradle/gradle-6.9.2/bin/gradle".freeze
-
   def gradle_dependencies
     dependency_metadata_regex = /-\s(?<group_id>.+):(?<artifact_id>.+):(?<version>.+)/
 
     # 'gradle dependencies' command needs to be run in the folder where buid.gradle is present.
-    shell_result = run_shell("#{GRADLE7} dependencies")
-
-    shell_result = run_shell("#{GRADLE6} dependencies") if !shell_result.success?
+    shell_result = run_shell("./gradlew dependencies")
```

Comment:
Is `gradlew` installed by docker?

Reply:
Is `gradlew` a file included in every gradle repo?

Reply:
yes, gradlew is a file included in every gradle project

Reply:
`spec/fixtures/gradle_wrapper/gradle/wrapper/gradle-wrapper.jar` contains the code that installs the gradle version specified in `spec/fixtures/gradle_wrapper/gradle/wrapper/gradle-wrapper.properties`

Reply:
Does it make sense to check if `gradlew` exists?

Reply:
yea it does, i've added a check for it